### DISCUSSION TITLE
Housekeeping - Segunda semana

### DISCUSSION THREAD

**Discussion by ernestoresende** [thtsystems/open-visit#9] [Announcements]
Olá pessoal.

Semana passada criamos a base do nosso sistema de autenticação, e movemos com sucesso na estrutura de banco de dados para o Postgres. O erro que encontramos no finalzinho da primeira sessão era relativamente simples de resolver:

https://github.com/thtsystems/open-visit/blob/045cd71d129c6c4f6015799a459dd58f1b38d117/auth/index.ts#L122-L134

A razão pela qual não tínhamos resposta ao executar nossa função é porque... de fato ela não respondia com nada 😅. Com isso resolvido as rotas de autenticação que configuramos devem funcionar perfeitamente desde que o database também esteja configurado corretamente.

Durante o final de semana, o @HenriqueStocco abriu um issue que nos fez ter um pequeno debate sobre as estranhezas dos sistemas de módulos do ecossistema do JavaScript. Pra quem tiver interesse de conhecer um pouco mais das entranhas do sistema de resolução de módulos, vale a leitura:
- #6 

Por último, foram abertos dois issues:
- #7 
- #8 

Admito que eles podem ser um pouco intimidantes visto que estão diretamente ligados à ferramenta de database que vamos usar, e o entendimento desta requer um pouco de leitura da documentação. Qualquer tentativa de resolvê-las, total, ou parcial, é válida.

Continuamos nesta quarta-feira na tentativa de criar alguns dos endpoints do backend, pra exemplificar o uso do Hono e como pensamos na estruturação de rotas em API's REST. E preparem-se... são muitos endpoints necessários pra cobrir a aplicação inteira. O que significa que não teremos tempo de terminar todos na quarta... O que significa que haverá vários issues pra que vocês possam implementá-los  👀

**Comment by CalebeBirer** [comment]
![image](https://github.com/thtsystems/open-visit/assets/90878309/61fcdc66-809b-4e4e-ae26-e5ec20155bf4)

good morning, The project on my machine has some errors, which I need to install initially, thanks.

**Reply by ernestoresende** [reply]
Did you run `pnpm install` in the root directory of the project? It seems it isn't finding the modules from the `node_modules` that should be on the `auth` workspace. If the issue remains, open an issue on the `Issues` tab on GitHub so we can try debugging it.
